app/src/main/java/org/kiwix/kiwixmobile/history/HistoryActivity.java

```diff
@@ -111,15 +116,32 @@ protected void onCreate(Bundle savedInstanceState) {
       actionBar.setTitle(R.string.history);
     }
 
-
     historyAdapter = new HistoryAdapter(historyList, deleteList, this);
     recyclerView.setAdapter(historyAdapter);
+    layoutManager = new LinearLayoutManager(this, RecyclerView.VERTICAL, false);
```

Comment:
We already have defined the layout manager in the XML. Instead of creating a new object, you can get it using `recyclerView.getLayoutManager()`.

Reply:
> 
> 
> We already have defined the layout manager in the XML. Instead of creating a new object, you can get it using `recyclerView.getLayoutManager()`.

@abdulwd  I forgot to see, from next time I will take care. Thanks for pointing it out.